client/src/main/java/io/pravega/client/stream/impl/ReaderGroupStateManager.java

```diff
@@ -330,6 +337,21 @@ private void fetchUpdatesIfNeeded() {
             compactIfNeeded();
         }
     }
+
+    void updateConfigIfNeeded() {
+        sync.fetchUpdates();
+        ReaderGroupState state = sync.getState();
+        if (!updateConfigTimer.hasRemaining() && state.isUpdatingConfig()) {
+            log.debug("Update the group config");
```

Comment:
Be more specific this won't actually help anyone debug a problem if there is one. 
Which reader group? What config? Is an update actually being performed?

Reply:
Made the debug statement more informative.